Documentation/contributing/making-changes.rst

```diff
@@ -211,15 +211,28 @@ squash before submitting the Pull Request:
    In the commit title please include the subsystem/area related to your contribution,
    followed by a descriptive message. Some examples:
 
+    Adding or fixing a platform
+
+    .. code-block:: bash
+
+       arch/<arch>/<platform>/: Add arch support for xxxxx platform
+
+       This patch adds initial support for xxxx platform. Please read
+       the documentation included for more details how to wire the display.
+
+       Signed-off-by: Your Name <you@whoareyou.com>
+
+    Adding or fixing a board
+
     .. code-block:: bash
 
-       boards/stm32f4discover: Add board initialization for SSD1306 OLED Display
+       boards/<arch>/<platform>/<board>: Add board initialization for SSD1306 OLED Display
```

Comment:
@jerpelea Thank you for improvements, some small consideration:

Normally the comments after board doesn't include the complete path, it is usually: boards/<boardname> as you can find in our git log history.

Also I think it is important to show a "real" example with a board name, so maybe include the information that should be "boards/<boardname>  and then show the example as it was initially.

Reply:
Why to avoid using complete path? Because the git log title is just 50 chars long, if we force a long prefix there will not enough space for info in the title.

Reply:
@acassis 
We need the complete path to be able to sort the commits for the release
board/architecture/platform/board_name
arch/architecture/platform
Without those changes sorting can not be optimised

Reply:
@jerpelea isn't the board unique for each architecture/platform? It is possible to retrieve the arch and plat from the board name.

Maybe we can follow Linux:
https://github.com/torvalds/linux/commits/master/arch/arm/configs/aspeed_g5_defconfig?after=786c8248dbd33a5a7a07f7c6e55a7bfc68d2ca48+34

Instead of using boards/boardname, use just use: \<arch\>/\<boardname\>:

Reply:
this will make things easy if we properly review

Reply:
we should be careful not to merge PRS that contain both arch, board, drivers and fixes commits
ex:
add new xxx board 

modify arch to support board
add initial board support
add xxx driver for the board
fix crash during boot